.gitlab-ci.yml

```diff
@@ -66,6 +68,14 @@ code_navigation:
   script:
     - go version
     - export PATH=$GOPATH/bin:$PATH
+    - export GIT_CLONE_PATH="$GOPATH/src/github.com/zaquestion/lab"
+    - export LAB_CORE_HOST="https://gitlab.com"
+    - export LAB_CORE_USER="lab-testing"
+    - export LAB_CORE_TOKEN="uDz6czYV412zK-xC5mUu"
+    - export CI_PROJECT_URL=$LAB_CORE_HOST
+    - export GITLAB_USER_LOGIN=$LAB_CORE_USER
+    - export CI_JOB_TOKEN=$LAB_CORE_TOKEN
+    - export LAB_CORE_TEST="GITLAB"
```

Comment:
If these are set in the `variables:` section of the ci file, do we really need to `export` them here again?

Reply:
It appears so.  If I do not export them the tests will run as 'prarit' instead of 'lab-testing', etc.

Reply:
That's sad. But ok :) nothing to worry about then.